### PR TITLE
Update documentation for Zsh

### PR DESCRIPTION
**Pull request by gupta-arpan**
This PR updates  _Installing-Oppia-(Linux;-Python-3).md_ to install Oppia on Linux systems using **Zsh**

### CODE REVIEW

**Comment by gupta-arpan** [comment]
@gp201 @SanjaySajuJacob  PTAL: Could you please review this PR and suggest some changes if required. Thanks

**Comment by Ash-2k3** [comment]
@U8NWXD The changes are related to installation guide, can you PTAL ? Thanks.

File: Installing-Oppia-(Linux;-Python-3).md [hunk 1]
```diff
@@ -111,7 +111,7 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ...
    ```
 
-   If you see the warning at the end, add the following lines to your `.bashrc` (see [here](https://askubuntu.com/a/127059) for where to find this file):
+   If you see the warning at the end, add the following lines to your `.bashrc` or `.zshrc` if you are using **Zsh** (see [here](https://askubuntu.com/a/127059) for where to find these files):
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]: The askubuntu link only refers to `.bashrc`, so maybe instead, let's just replace `.bashrc` with `~/.bashrc` and `.zshrc` with `~/.zshrc` to make it clear where these files are

File: Installing-Oppia-(Linux;-Python-3).md [hunk 2]
```diff
@@ -149,7 +149,11 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ```bash
    eval "$(direnv hook bash)"
    ```
+   If you are using **Zsh**, add following lines to the end of `.zshrc`
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md]:
```suggestion

   If you are using **Zsh**, add following lines to the end of `.zshrc`:
```

File: Installing-Oppia-(Linux;-Python-3).md [hunk 3]
```diff
@@ -149,7 +149,11 @@ For your virtual environment, we recommend you use [pyenv](https://github.com/py
    ```bash
```
Comment (U8NWXD) [inline Installing-Oppia-(Linux;-Python-3).md:150]: Like above, can you replace `.bashrc` with `~/.bashrc` and drop the askubuntu link? Also, please make clear that this change only applies if one is using bash (e.g. add "If you are using Bash, ...")

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Installing-Oppia-(Linux;-Python-3).md (hunk 3 above)
Comment (gupta-arpan) [reply Installing-Oppia-(Linux;-Python-3).md:150]: Done, kindly review it.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**